tests/main/upgrade-from-release/task.yaml

```diff
@@ -48,7 +48,7 @@ execute: |
     fi
 
     declare -A EXPECTED_SNAPD_VERSIONS=(
```

Comment:
Thanks for the fix. I think longer term we need to find a way to auto-update expected list.